app/models/review.rb

```diff
@@ -54,8 +54,9 @@ class Review < ApplicationRecord
   scope :order_by_created_at,      -> { order(created_at: :desc) }
   scope :by_reviewer,              -> (reviewer)   { where(reviewer_id: reviewer.id) }
   scope :by_submission,            -> (submission) { where(grant_submission_submission_id: submission.id) }
-  scope :completed,                -> { where.not(overall_impact_score: nil)}
-  scope :incomplete,               -> { where(overall_impact_score: nil)}
+  scope :completed,                -> { where.not(overall_impact_score: nil) }
+  scope :incomplete,               -> { where(overall_impact_score: nil) }
+  scope :may_be_reminded,          -> { incomplete.where("reminded_at IS NULL OR reminded_at < ?", 1.week.ago) }
```

Comment:
is this scope being used somewhere? it probably should be used to keep from repeated reminders being sent

Reply:
currently not being used, no. Reminder throttling was not a request. Should be removed.

Reply:
i think it's fine to leave it here maybe add some comment indicating TODO